src/node/consumer/src/repository/models/content.ts

```diff
@@ -0,0 +1,22 @@
+import {Table, Column, Model, HasMany, DataType, BelongsTo, ForeignKey} from 'sequelize-typescript';
+import {ContentAttributes, ContentCreationAttributes} from "../interfaces/content_attributes";
+import {Torrent} from "./torrent";
+
+@Table({modelName: 'content', timestamps: false})
+export class Content extends Model<ContentAttributes, ContentCreationAttributes> {
+    @Column({ type: DataType.STRING(64), primaryKey: true, allowNull: false, onDelete: 'CASCADE' })
+    @ForeignKey(() => Torrent)
+    declare infoHash: string;
```

Comment:
What's `declare` for?

Reply:
declare has to be using when using Sequelize in typescript, because otherwise the properties mask the properties sequelize generate

From here: https://sequelize.org/docs/v6/other-topics/typescript/